app/controllers/v0/burial_claims_controller.rb

```diff
@@ -1,37 +1,35 @@
 # frozen_string_literal: true
 
 require 'pension_burial/tag_sentry'
+require 'burials/monitor'
 
 module V0
   class BurialClaimsController < ClaimsBaseController
     service_tag 'burial-application'
 
     def show
-      submission_attempt = determine_submission_attempt
+      claim = claim_class.find_by!(guid: params[:id])
+      form_submission = claim&.form_submissions&.last
+      submission_attempt = form_submission&.form_submission_attempts&.last
       if submission_attempt
         state = submission_attempt.aasm_state == 'failure' ? 'failure' : 'success'
         render(json: { data: { attributes: { state: } } })
       elsif central_mail_submission
         render json: CentralMailSubmissionSerializer.new(central_mail_submission)
-      else
-        Rails.logger.error("ActiveRecord::RecordNotFound: Claim submission not found for claim_id: #{params[:id]}")
-        render(json: { data: { attributes: { state: 'not found' } } }, status: :not_found)
       end
+    rescue ActiveRecord::RecordNotFound => e
+      monitor.track_show404(params[:id], current_user, e)
+      render(json: { data: { attributes: { state: 'not found' } } }, status: :not_found)
     rescue => e
-      Rails.logger.error(e.to_s)
+      monitor.track_show_error(params[:id], current_user, e)
       render(json: { data: { attributes: { state: 'error processing request' } } }, status: :unprocessable_entity)
     end
 
-    def create
+    def create # rubocop:disable Metrics/MethodLength
```

Comment:
could the rubocop exception be eliminated if moving the 'logging' for unsuccessful save to a function

Reply:
also would want to add something similar to `log_validation_error_to_metadata` that is in the pension controller